NuGet.config

```diff
@@ -12,6 +12,7 @@
       one as a template. The following line is a marker to insert the test restore sources.
     -->
     <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
+    <add key="vihofer-staging" value="https://www.myget.org/F/vihofer-staging/api/v3/index.json" />
```

Comment:
Note that the second commit in this PR which updates this file and the Versions.props file will be reverted before merging. This is just temporary to consume the arcade change and test it out in CI.